coursedata/texts/en.yaml

```diff
@@ -153,6 +153,10 @@ Auth:
     other_text: "Another text language"
     teacher_invitation_require_login: "To set up your profile as a teacher we will need you to log in. If you don't have an account, please create one."
     teacher_welcome: "Welcome to Hedy! Your are now the proud owner of a teachers account which allows you to create classes and invite students."
+    example_left: "You can still try 1 example program"
```

Comment:
I think we can remove the texts for x left if we only use the modal?

Reply:
Yes, this these can be removed!